src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.searchBar {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-size: 16px;
+  line-height: 22px;
+  padding: 24px 62px;
+}
+
+.smallSearchBar {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding: 11px 33px;
+}
+
+.image {
+  position: absolute;
+  top: 25px;
+  left: 25px;
+}
+
+.imageSmall {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+}
```

Comment:
This two classes has duplicated styles, create on class for big image and one for small (where you will only overwrite size values) then add two classes to small image